client.go

```diff
@@ -459,8 +651,8 @@ func (c *Client) runRepliesConsumer(inChan *amqp.Channel) error {
 		c.consumeSettings.Consumer,
 		c.consumeSettings.AutoAck,
 		c.consumeSettings.Exclusive,
-		c.consumeSettings.NoLocal,
-		c.consumeSettings.NoWait,
+		false, // no-local.
```

Comment:
See above, I don't get why these are removed (they still seem to be the same everywhere they're hard coded).